test/e2e/elasticache/smoke_cache_subnet_group.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```

Comment:
Go ahead and rename this file to `test/e2e/elasticache/cache_subnet_group/smoke.sh` and we'll put all the cache subnet group tests in a separate directory. The e2e bash test runner handles this just fine. For the ECR and S3 smoke tests, it's fine to have just a single `smoke.test` file in the main directory since there's only one resource in those APIs (Repository and Bucket, respectively).

We will eventually create a `test/e2e/elasticache/smoke.sh` test file that will test related/dependent object creation, but for these single-resource tests, let's place those in separate directories associated with the specific resource. Sound good?

Reply:
Updated the file per the comment.